packages/cli/package.json

```diff
@@ -7,7 +7,8 @@
   "type": "module",
   "license": "MIT",
   "bin": {
-    "mud": "./dist/mud.js"
+    "mud": "./dist/mud.js",
+    "mud2": "./dist/mud2.js"
```

Comment:
`mud2` without v2 commands is confusing, what about `mud-v1` or `mud-deprecated`?

Reply:
`mud2` is _exclusively_ v2 commands

Reply:
ohhh, the PR title was misleading!

Reply:
oops true! good catch